src/App.js

```diff
@@ -1,9 +1,11 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import FreakTile from './components/FreakTile/FreakTile';
 import { freaks } from './mock-data/freaks.json';
+import Modal from './components/Modal/Modal';
 
 function App() {
+  const [ isOpen, setOpen ] = useState(false);
```

Comment:
I think the SET function should always contain the full name of the variable it is setting, so instead of `[ isOpen, setOpen ]` it should be `[ isOpen, setIsOpen ]`.